clusters/app.ci/cert-manager/cluster-bot-ingress.yaml

```diff
@@ -0,0 +1,23 @@
+kind: Ingress
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: ci-chat-bot
+  namespace: ci
+  annotations:
+    cert-manager.io/cluster-issuer: cert-issuer
+    haproxy.router.openshift.io/timeout: 2m
+spec:
+  tls:
+    - hosts:
+        - clusterbot.ci.openshift.org
+      secretName: clusterbot-tls
+  rules:
+    - host: clusterbot.ci.openshift.org
+      http:
+        paths:
+          - backend:
+              service:
+                name: ci-chat-bot
+                port:
+                  number: 8080
+            pathType: ImplementationSpecific
```

Comment:
Just an FYI, an `Ingress` isn't really essential to this process, it really just gives us a "nice" url to use (clusterbot.ci.openshift.org) versus the default name (like ci-chat-bot-ci.apps.ci.l2s4.p1.openshiftapps.com).  
Technically, both will work exactly the same way.